src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -118,6 +118,102 @@ public void SarifTransformerTests_ToCurrent_MinimumWithTwoRuns()
             v2LogText.Should().Be(v2LogExpectedText);
         }
 
+        [Fact]
+        public void SarifTransformerTests_ToCurrent_MinimumWithPropertyAndTags()
+        {
+            string v1LogText =
+              @"{
+                  ""version"": ""1.0.0"",
+                  ""runs"": [
+                    {
+                      ""tool"": {
+                        ""name"": ""CodeScanner"",
+                        ""semanticVersion"": ""2.1.0"",
+                        ""properties"": {
+                          ""foo"": ""bar"",
+                          ""tags"": [ ""1"", ""2"" ]
+                        }
+                      },
+                      ""results"": []
+                    }
+                  ]
+                }";
+
+            SarifLog v2Log = TransformVersionOneToCurrent(v1LogText);
+
+            string v2LogText = JsonConvert.SerializeObject(v2Log, SarifTransformerUtilities.JsonSettingsV2);
+            string v2LogExpectedText =
+@"{
+  ""$schema"": ""http://json.schemastore.org/sarif-2.0.0"",
+  ""version"": ""2.0.0"",
+  ""runs"": [
+    {
+      ""tool"": {
+        ""name"": ""CodeScanner"",
+        ""semanticVersion"": ""2.1.0"",
+        ""properties"": {
+          ""foo"": ""bar"",
+          ""tags"": [
+  ""1"",
+  ""2""
+]
+        }
+      },
+      ""results"": []
+    }
+  ]
+}";
+
+            v2LogText.Should().Be(v2LogExpectedText);
+        }
+
+        [Fact]
+        public void SarifTransformerTests_ToCurrent_MinimumWithTagsOnly()
```

Comment:
I don't see the added value of this test unless we're worried that tags only come over if some _other_ property also comes over. For me the first test is enough.

Reply:
Fine by me. I tend to err on the side of excess with tests